lib/security.js

```diff
@@ -70,7 +70,7 @@ function getSecurityConfig (app, forceRead = false) {
       return JSON.parse(optionsAsString)
     } catch (e) {
       console.error('Could not parse security config')
-      console.error(e)
+      console.error(e.message)
```

Comment:
Why this change? I like the previous way because you get a stack trace along with the message.

Reply:
To get rid of the stack trace ;-). I did not like something pretty ok, like unconfigured security, throwing stacktraces. The previous line's error message pinpoints where in code the error happens and there are only so many ways the code block in question can fail.

It could be that I am doing the wrong thing here and we should test if security is configured in some other way so that no stacktraces are flying around.